sensor_array/simulation/write_simulations.py

```diff
@@ -4,21 +4,25 @@
 # ----- Import Python Packages -----
 # ----------------------------------
 import csv
-import os
-import sys
 from datetime import datetime
-
 import numpy as np
+import os
+import shutil
 import sjs
-
 import subprocess
-import shutil
+import sys
```

Comment:
pep8:

Imports should be grouped in the following order:

    Standard library imports.
    Related third party imports.
    Local application/library specific imports.

You should put a blank line between each group of imports.